src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchFilterTest.java

```diff
@@ -259,6 +259,114 @@ public void testFileTabCharacter() throws Exception {
+                "strategy/FileTabCharacter/MoveCodeInSameFileWithoutChanges/"
+                        + "patchedline/zeroContextConfig.xml";
+        final String[] expectedTwo = {
+            "19:25: Line contains a tab character.",
+        };
+        testByConfig(defaultContextConfigPathTwo, inputFile, expectedTwo);
+        testByConfig(zeroContextConfigPathTwo, inputFile, expectedTwo);
+    }
+
+    @Test
+    public void testFileTabCharacterMoveCodeInSameFileWithMinorChanges() throws Exception {
+        final String inputFile =
+                "strategy/FileTabCharacter/MoveCodeInSameFileWithMinorChanges/Test.java";
+
+        final String defaultContextConfigPathOne =
+                "strategy/FileTabCharacter/"
+                        + "MoveCodeInSameFileWithMinorChanges/newline/defaultContextConfig.xml";
+        final String zeroContextConfigPathOne =
+                "strategy/FileTabCharacter/"
+                        + "MoveCodeInSameFileWithMinorChanges/newline/zeroContextConfig.xml";
+        final String[] expectedOne = {
+            "19:25: Line contains a tab character.",
+        };
+        testByConfig(defaultContextConfigPathOne, inputFile, expectedOne);
+        testByConfig(zeroContextConfigPathOne, inputFile, expectedOne);
+
+        final String defaultContextConfigPathTwo =
+                "strategy/FileTabCharacter/MoveCodeInSameFileWithMinorChanges"
+                        + "/patchedline/defaultContextConfig.xml";
+        final String zeroContextConfigPathTwo =
+                "strategy/FileTabCharacter/MoveCodeInSameFileWithMinorChanges"
+                        + "/patchedline/zeroContextConfig.xml";
+        final String[] expectedTwo = {
+            "19:25: Line contains a tab character.",
+        };
+        testByConfig(defaultContextConfigPathTwo, inputFile, expectedTwo);
+        testByConfig(zeroContextConfigPathTwo, inputFile, expectedTwo);
+    }
+
+    @Test
+    public void testFileTabCharacterSingleChangedLine() throws Exception {
+        final String inputFile = "strategy/FileTabCharacter/SingleChangedLine/Test.java";
+
+        final String defaultContextConfigPathOne =
+                "strategy/FileTabCharacter/SingleChangedLine/newline/defaultContextConfig.xml";
+        final String zeroContextConfigPathOne =
+                "strategy/FileTabCharacter/SingleChangedLine/newline/zeroContextConfig.xml";
+        final String[] expectedOne = {};
+        testByConfig(defaultContextConfigPathOne, inputFile, expectedOne);
+        testByConfig(zeroContextConfigPathOne, inputFile, expectedOne);
+
+        final String defaultContextConfigPathTwo =
+                "strategy/FileTabCharacter/SingleChangedLine/patchedline/defaultContextConfig.xml";
+        final String zeroContextConfigPathTwo =
+                "strategy/FileTabCharacter/SingleChangedLine/patchedline/zeroContextConfig.xml";
+        final String[] expectedTwo = {
+            "19:25: Line contains a tab character.",
```

Comment:
I do not understand why this is a violation, moved to https://github.com/checkstyle/patch-filters/issues/103